### ISSUE TITLE
Examples no training progress

### ISSUE THREAD

**Issue by Kalkasas** [ppwwyyxx/tensorpack#8]
I setup tensorpack and i can run all the examples. But the training shows no progress at all. For Cifar 10 and mnist the training and validation error doesnt change at all over multiple epochs.

I use tensorflow 0.9 and a GTX 1080. A hint of any kind would be helpfull. I like the architecture of the tensorpack and i would like to use it.

**Comment by ppwwyyxx** [comment]
Hi,
I'm not sure what's going wrong. But I tested on 1080 before and did see some weird things on that.
Now I don't have a 1080 at hand for testing, but maybe you could run on CPU to see if things go well. For mnist it should get 98~99% accuracy after first epoch. 

I'm settings `CUDA_VISIBLE_DEVICES` in `mnist-convnet.py` to 0 by default, so maybe you'll need to delete that line to run on CPU.

**Comment by Kalkasas** [comment]
I can reproduce your results on the CPU of the tensorpack example. I can also reproduce the official mnist tensorflow results with GPU.

I am using the older cuda and cudnn versions which are compatible with the published tensorflow binaries. Maybe compiling from source with the latest version can help?

**Comment by ppwwyyxx** [comment]
Maybe my code uses some part of tensorflow that's broken on 1080. I may get some access to test on 1080 in a week but not now.

Also, I think only cuda 8 supports Pascal Architecture. How can you use 1080 with older cuda versions?

**Comment by Kalkasas** [comment]
The issue seems to be resolved by updaten drivers and cuda versions. Thank you for your help!
